_episodes/39-vodik.md

```diff
@@ -0,0 +1,39 @@
+---
+layout:     episode
+title:      "Vodík v souvislostech"
+number:     39
+slug:       "39-vodik"
+redirect_from: "/epizody/39"
+published:  2023-02-17
+summary:    |
+   Přestože se o využití vodíku mluví v mnoha odvětvích, ne vždy se musí jednat o ten nejlevnější způsob dekarbonizace. V této epizodě proto s Martinem Tenglerem (Bloomberg NEF) diskutujeme nejen technologický potenciál vodíku a jeho využití v průmyslu, ale hlavně ekonomické souvislosti výroby a přepravy. Ty totiž do značné míry ovlivní jeho možnou budoucnost ve světové ekonomice.
+audioboom:  "8248988"
+guests: [ tengler ]
+transcript: false
+---
+Už dnes ovšem celosvětově spotřebováváme za rok tolik energie z vodíku, kolik jí spotřebuje USA ze zemního plynu za půl roku. Vodík tedy stojí za bližší seznámení, proto probereme jeho nejzajímavější fyzikální a chemické vlastnosti a vysvětlíme vám, jak se vyrábí dnes nebo jak by se mohl produkovat za deset let. S tím souvisí i diskuze, jak se vodíku daří na politické a legislativní úrovni nejen v Evropské unii.
+
+Epizoda vznikla za podpory Britského velvyslanectví v Praze.
+
+{% include 2050/guest-list.html %}
+
+---
+
+## Bonusové materiály
+
+<div class="bonus-material" markdown="1">
+
+{% include 2050/bonus-interview.html
+  title="Martin Tengler: O využití vodíku nejen v průmyslu v ekonomických souvislostech"
+  audioboom="8248966"
+  spotify="16rbtISczort6F5j5GKRjx"
+  caption="Prodloužená verze 39. epizody podcastu, která obsahuje dodatečné informace k historii využití vodíku, výrobě vodíku a jeho využití v dekarbonizaci lodní a letecké dopravy"
+%}
+
+</div>
+
+## Transkript epizody
+
+---
```

Comment:
Obvykle máš tu oddělovací čáru nad nadpisem, je v tom záměr, že je to tady naopak?

Současně mě zpětně překvapuje, že ty čáry tam vkládáš ručně. To by se dalo nastavit, že každý H2 u epizody má nad sebou oddělovací čáru... (to ale v tomto PR neřešme)

Reply:
nene, záměr není - já pořád moc nevím, jak ty čáry fungují..